lib/measured/units/weight.rb

```diff
@@ -1,6 +1,10 @@
 Measured::Weight = Measured.build do
   unit :g, aliases: [:gram, :grams]
   unit :kg, value: "1000 g", aliases: [:kilogram, :kilograms]
+  unit :T, value: "1000 kg", aliases: [:tonne, :tonnes]
+  unit :slug, value: "14.593903 kg", aliases: [:slugs]
+  unit :N, value: "9.80665 kg", aliases: [:newtons, :newton]
+  unit :t, value: "2000 lb", aliases: [:ton, :tons]
```

Comment:
My only issue here is that after reading https://en.wikipedia.org/wiki/Short_ton, there is some ambiguity in the term "ton". I think our best bet is just using the long form of "short ton" and "long ton".

Also, https://en.wikipedia.org/wiki/Tonne is telling me that `t` is actually used for the metric ton, or what you have as `tonne` above. It doesn't look like a short ton actually has a shortened (pun unintended!) form.

Finally, perhaps we should add long ton too :)